frontend/src/components/resetPassword.jsx

```diff
@@ -22,13 +27,55 @@ export default function ResetPassword() {
     }
   };
 
+  const updateAWS = async () => {
+    // const verificationCode = await getCode();
+    const password = password2;
+    const user = new CognitoUser({
+      Username: localStorage.getItem('woods-humane-email'),
+      Pool: userPool,
+    });
+
+    user.confirmPassword(code, password, {
+      onSuccess: () => {
+        console.log('Password confirmed!');
+      },
+      onFailure: (err) => {
+        console.log('Password not confirmed!');
```

Comment:
If something goes wrong let's prompt the user so they at least know there's an issue